libs/net/dal/Migrations/1.0.15/Up/PostUp/00-NotificationRazorTemplate.sql

```diff
@@ -0,0 +1,80 @@
+DO $$
+BEGIN
+
+UPDATE public."notification"
+SET
+	settings=regexp_replace('{ "subject": "@model TNO.Services.Notification.Models.TemplateModel\n
```

Comment:
@CelloLiu This template would have never worked without proper @model and @using statements.  Additionally, Entityframework could not work with the formatting your original script entered into the database.